pyaerocom/test/test_colocation_auto.py

```diff
@@ -0,0 +1,181 @@
+import os
+import pytest
+
+from pyaerocom.conftest import TESTDATADIR
+from pyaerocom.conftest import does_not_raise_exception
+from pyaerocom.conftest import testdata_unavail
+
+from pyaerocom import Colocator, ColocatedData, GriddedData, UngriddedData
+from pyaerocom.io import ReadGridded
+from pyaerocom.exceptions import DataCoverageError
+from pyaerocom.io.aux_read_cubes import add_cubes
+
+
+@pytest.fixture(scope='module')
+def init_obsdata_paths():
+    import pyaerocom as pya
+    pya.const.add_data_search_dir(TESTDATADIR.joinpath('obsdata'))
+
+@pytest.fixture(scope='function')
+def col_tm5_aero(model_id='TM5-met2010_CTRL-TEST', obs_id='AeronetSunV3L2Subset.daily',
+                 obs_vars='od550aer'):
+    coloc = Colocator(model_id=model_id, obs_id=obs_id, obs_vars=obs_vars)
+    coloc.raise_exceptions = True
+    coloc.reanalyse_existing = True
+    coloc.start = 2010
+    return coloc
```

Comment:
Using `scope='function'` seems to be an interesting concept that I have not used before. I trust you that this makes sense, as I do not have the time right now to really understand why you do it this way.

Reply:
This is done since some of the tests might change the properties of coloc.
Should be okay since the fixture is not doing anything resource intensive.